custom_components/ha_washdata/frontend.py

```
@@ -379,23 +373,66 @@ async def async_register_panel(hass: HomeAssistant) -> bool:
+    # result immediately, so later callers (e.g. a second device added after
+    # boot) are fast no-ops.
+    if PANEL_TASK_KEY not in hass.data:
+        hass.data[PANEL_TASK_KEY] = hass.async_create_task(
+            _do_register_panel(hass, src)
+        )
+
+    try:
+        return bool(await hass.data[PANEL_TASK_KEY])
+    except asyncio.CancelledError:
+        return False
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)